src/components/picker/PickerModal.js

```diff
@@ -89,7 +89,7 @@ class PickerModal extends BaseComponent {
         visible={visible}
         onRequestClose={topBarProps.onCancel}
         onShow={onShow}
-        overlayBackgroundColor={overlayBackgroundColor}
+        overlayBackgroundColor={pickerModalProps.backgroundColor}
```

Comment:
Better spread all of the props, otherwise you just support `overlayBackgroundColor`
```suggestion
        {...pickerModalProps}
```